### DISCUSSION TITLE
Allow users to label their issues

### DISCUSSION THREAD

**Discussion by toasterovenly** [Emplexx/Aluminium#25] [Ideas]
When making new GitHub issues today, I ended up following the pattern other issues were using and putting "[Feature Request]" at the beginning of many of the issues since they weren't bug reports. (Although thinking about it now, maybe many weren't actually full features either...) I would prefer to just add a "feature request", "bug", or "request" label to each of my GitHub issues instead of prefixing the issue name.

I just looked it up and I think I should be able to add any existing label to an issue, even without write access to the repo. You definitely need write access to modify the existing labels. Are there just no labels yet in this repo? I couldn't seem to add any to the issues I was authoring.

**Comment by Emplexx** [comment]
That's weird, this repo has the default set of labels already I'm pretty sure

**Comment by toasterovenly** [comment]
I did some more research. For a repo owned by an individual, the access levels are `owner` and `collaborator`. Both of those can manage and apply labels. It seems everyone who isn't explicitly added to the repo can't do anything involving labels. For organizations, a user only needs `triage` level access, which is the second lowest permission level above `read` permissions.

So I guess this is juts how GitHub works. I am so used to being in an org that I didn't realize I wouldn't be able to add labels to my own issues.

**Comment by Emplexx** [comment]
Ah, thank you for looking into it! Guess I just gotta tag them manually
